modules/priors/tests/Rcheck_reference.log

```diff
@@ -118,60 +105,22 @@ priorfig: no visible binding for global variable ‘x’
 priorfig: no visible global function definition for ‘runif’
 priorfig: no visible binding for global variable ‘y’
 Undefined global functions or variables:
-  AIC density last plot_density qbeta qgamma qlnorm qnorm qweibull
-  runif tabnum x y zero.bounded.density
+  AIC last plot_density qbeta qgamma qlnorm qnorm qweibull
```

Comment:
AIC and all of these q* and r* distributions are in stats::

Reply:
Fixed in 6f15a88c1